e2e-tests/cypress/tests/integration/channels/channel/convert_group_message_to_private_spec.js

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
+// Group: @channels @channel
+
+describe('Group Message', () => {
```

Comment:
Wonder if it makes sense to handle any of the "error cases" (mainly the API not working? not sure what else can error here)

Reply:
Not needed IMO as we don't necessarily handle all cases in e2e. Unit tests are fine.